noxfile.py

```diff
@@ -475,14 +475,14 @@ def set_minimal_package_versions(session: nox.Session):
     min_pkg_versions = {
         "rust_decimal": "1.26.1",
         "csv": "1.1.6",
+        "indexmap": "1.9.3",
```

Comment:
`hashbrown` depends on `indexmap`, I was having trouble locally setting minimal package versions on 1.56 without putting `indexmap` above it.